src/Components/Publishing/Display/__tests__/DisplayCanvas.test.tsx

```diff
@@ -131,6 +132,10 @@ describe("unit", () => {
     ])
   })
 
+  it("checks for enabled ads", () => {
+    expect(isHTLAdEnabled()).toBe(false)
+  })
```

Comment:
Rather than test inside of `DisplayCanvas.test`, it would be better to create a `isHTLAdEnabled.test` file and test expectations there. (Which will also allow for the testing of a `toBe(true)` case.)